.github/workflows/release-pkg.nu

```diff
@@ -128,7 +129,7 @@ sleep 3sec
 
 print $'(char nl)Check binary release version detail:'; hr-line
 let ver = if $os == 'windows-latest' {
-    (do -i { ./output/nu.exe -c 'version' }) | str join
+    (do -i { .\output\nu.exe -c 'version' }) | str join
```

Comment:
Was this not working before?

Reply:
It works before, but it's the right way to do it on Windows, so it's kept.